app/pages/onboarding/08-set-password/set-password.tsx

```diff
@@ -1,6 +1,8 @@
 import React, { useState } from 'react';
+import { useHistory } from 'react-router';
 import { Input } from '@blockstack/ui';
 
+import routes from '../../../constants/routes.json';
```

Comment:
It would be nice to get some TS paths setup.

Reply:
Will get around to this, maybe not in this PR.